docs/docs/gpus.md

```diff
@@ -0,0 +1,69 @@
+# Preferential GPU Scheduling
+
+## Overview of GPU Scheduling in Metronome
+
+In order to have Metronome run jobs with GPU workloads it is necessary to:
+
+1. Understand the Mesos configuration for GPU management.
+2. Start Metronome with GPU behavior (`METRONOME_GPU_SCHEDULING_BEHAVIOR` detailed below).
+3. Specify gpus resource using whole numbers only.
+4. Use the UCR containerizer.
+
+
+Assuming that Mesos is configuration to filter GPU resources (details below), the following is what is necessary in metronome.
+
+## GPU Startup Parameters in Metronome
+
+Metronome supports launching GPU tasks when the environmental variable `METRONOME_GPU_SCHEDULING_BEHAVIOR` is specified.  If this env var is *not* specified, Metronome will not opt-in for receiving Mesos offers which contain GPU resources.  Depending on the GPU availability, it may be desirable to configure Metronome to avoid placing non-GPU tasks on GPU nodes. By *not* specifying GPU scheduling *and* while Mesos is configured to filter GPU resources, Metronome will *not* receive offers containing GPU resources.   When `METRONOME_GPU_SCHEDULING_BEHAVIOR` is specified with an appropriate option (listed below), then Metronome is opting in to receiving offers with GPUs.  Below is the options and behavior when specifying Metronome GPU scheduling behavior via the `METRONOME_GPU_SCHEDULING_BEHAVIOR` env variable when starting Metronome:
+
+  - `unrestricted` - non-GPU tasks are launched irrespective of offers containing GPUs.
+  - `restricted` - non-GPU tasks will decline offers containing GPUs with a decline reason of `DeclinedScareResources`.
+
+While Metronome is in `unrestricted` mode, it will match job run tasks without gpus defined on nodes with gpu resources if all other constraints are met.   While in `restricted` mode, a job run defined with a GPU requirement will only match an offer that has required GPUs in the offer and jobs without GPUs needs will never match offers with GPU (leaving those GPU resources for tasks that need them).
```

Comment:
>  it will match job run tasks

This was hard to grok on first 3 reads; do we need a hyphen maybe?

Reply:
Or maybe just remove the clarifying word "job run" in "job run tasks", since all tasks in metronome are "job run tasks"?

Reply:
I'd go for just saying "tasks"